bulk_query_gen/influxdb/influx_iot_light_level.go

```diff
@@ -0,0 +1,31 @@
+package influxdb
+
+import (
+	"time"
+
+	bulkQuerygen "github.com/influxdata/influxdb-comparisons/bulk_query_gen"
+)
+
+type InfluxIotLightLevel struct {
+	InfluxIot
+}
+
+func NewInfluxqlIotLightLevel(dbConfig bulkQuerygen.DatabaseConfig, queriesFullRange bulkQuerygen.TimeInterval, queryInterval time.Duration, scaleVar int) bulkQuerygen.QueryGenerator {
```

Comment:
Really minor nitpick, I've noticed we tend to do `InfluxQL` instead of `Influxql`

Reply:
Smashed merged before I saw this comment, but I agree - I'll try to remember to clean this up next time I do an edit here!